docker-compose.yml

```diff
@@ -15,7 +15,7 @@ services:
       # Default configuration for persist_directory in chromadb/config.py
       # Read more about deployments: https://docs.trychroma.com/deployment
       - chroma-data:/chroma/chroma
-    command: uvicorn chromadb.app:app --reload --workers 1 --host 0.0.0.0 --port 8000 --log-config chromadb/log_config.yml --timeout-keep-alive 30
+    command: "--workers 1 --host 0.0.0.0 --port 8002 --proxy-headers --log-config chromadb/log_config.yml --timeout-keep-alive 30"
```

Comment:
@MrZoidberg, why do you think this is an improvement over the existing command? I find the `unicorn` command much less confusing than a bunch of parameters, which the user needs to jump over to another file to understand what is happening.

Can you also revert the port to `8000`

Reply:
Reverted to 8000